### ISSUE TITLE
Map win panel requires team names to match exactly

### ISSUE THREAD

**Issue by vladxy** [drweissbrot/cs-hud#117]
When I use the bo3 setup, after the 1st map is set and the score is submitted, it shows on the HUD that the score is 1-1 instead of 1-0

**Comment by drweissbrot** [comment]
I'm not entirely sure what the situation is here. Could you post a screenshot of the `Series: Map 1` to `Series: Map 5` sections on the config page, or the `theme.json` from your `cs-hud/userspace` directory?

**Comment by vladxy** [comment]
> ![pEORTV7](https://private-user-images.githubusercontent.com/156136783/295587669-1a38b57a-b7da-495a-858e-2e7fdc70027c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ5Mjk4ODEsIm5iZiI6MTcwNDkyOTU4MSwicGF0aCI6Ii8xNTYxMzY3ODMvMjk1NTg3NjY5LTFhMzhiNTdhLWI3ZGEtNDk1YS04NThlLTJlN2ZkYzcwMDI3Yy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDExMFQyMzMzMDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05MTY5MGU4ZWRlZmE3ODRiNzE0NmI2N2QyYTA5ZWFiNDJkYjc0MTY0NmNiMDdmMmZmOTdhYWMzZjJkZjFlZjYxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.AsLT5AGgurRrpfm8LY_1ksyT-Umt0umRwkhzlK-Xi68) Those were the setting for Map 1 through 3, map 4 and 5 were left completely blank. With these scores, it showed as 1-1 on the hud under the timer. Should I post the theme.json as well?

Also want to add to this one, the score was not submitted to the second map when I encountered the error, when it was submitted like in this screenshot, it showed the score under the timer as "2-2"

**Comment by drweissbrot** [comment]
Interesting, I had a look at it with the demo (https://www.hltv.org/matches/2368847/9ine-vs-koi-pgl-cs2-major-copenhagen-2024-europe-rmr-open-qualifier-1) and the data from your screenshot, and it looks like the HUD's getting confused because the team names don't match exactly. The HUD thinks a team called `9INE` lost map 2, so when it finds a team that's not called `9INE`, it'll consider them a winner of that match -- but neither team is called `9INE` (because they're actually called `9INEgg` on the server), so it just gives them both a win.

I'm not immediately sure if it's possible to work around that in software, will think about it. Thanks for reporting!

**Workaround for now:** You can either just type `9INEgg` in `series.maps.2.pickTeam`, or you could rename `9INEgg` to `9INE`, cf. https://github.com/drweissbrot/cs-hud/blob/c8f8a93574fe74eb756e46a2d7f6f16a692d4559/docs/team-name-overrides.md.

**Comment by vladxy** [comment]
Will try in the bo3 and I'll get back to you

**Comment by vladxy** [comment]
So indeed that was the issue, it works if the names are exactly the same as the ones on the server, but the bo1 thing still doesn't work properly I think

**Comment by drweissbrot** [comment]
> Also kind of in the same field, when the match is setup as a bo1 and the checkmark for decider is not set, it shows the match as being a bo2, first map is showing up correctly but it also shows a blank "decider", and under the timer it reads "map 1 out of 2"

Could you make sure that the decider checkbox is not set for any map? If it's checked for map 3 for example, that'll cause that phantom map to show up.

**Comment by vladxy** [comment]
I redid the settings as for a bo1, only thing I put in was the map name for the 1st map, nothing else on any of them. With those settings, if it's supposed to show the map that is being played, nothing is showing.

**Comment by drweissbrot** [comment]
Correct, it's not supposed to show the "Map 1 of 1" bit below the clock, nor the map name on the bottom during freezetimes for a BO1.  
If you really want it, you can force it to show the current map during freezetimes by checking the `preferences.seriesGraph.showMapForOnlyMatch` checkbox.

**Comment by vladxy** [comment]
Okay, thanks for the help, so far the HUD as been awesome, great job!
